drill-admin/src/main/kotlin/com/epam/drill/endpoints/WsTopic.kt

```diff
@@ -75,3 +68,31 @@ class Temp<T, R>(val block: (R, MutableSet<DrillWsSession>) -> T) {
     }
 }
 
+@UseExperimental(ImplicitReflectionSerializer::class)
+fun serialize(value: Any?): String {
+    if (value == null) return ""
+    val serializer = when (value) {
+        is String -> null
```

Comment:
may be StringSerializer

Reply:
no, may not. it turns the origin string to total mess